### DISCUSSION TITLE
Add exclusions after installing via HACS

### DISCUSSION THREAD

**Discussion by allymparker** [alandtse/alexa_media_player#1228] [Q&A] [answered]
Is it possible to add device exclusions after installing via HACS?  I noticed there was an option during install, but when I choose options in integrations, I can't see any way to configure it?

**Comment by alandtse** [comment, marked as answer]
The exclusions are done during component install; it's not related to the HACS copying of the files. You can either [remove or readd](https://github.com/custom-components/alexa_media_player/wiki/Configuration#integration-page) on the integrations or use the built in HA exclusion options.
